server/parser/src/main/antlr4/org/eclipse/lsp/cobol/core/CobolParser.g4

```diff
@@ -2181,18 +2173,20 @@ length
    ;
 
 argument
-   : arithmeticExpression
-   | TRAILING | LEADING
+   : ALL
+   | arithmeticExpression
+   | TRAILING
+   | LEADING
    ;
```

Comment:
Looks like ALL is a valid argument as per  https://www.ibm.com/docs/en/cobol-zos/6.4?topic=reference-references-data-division-names and https://www.ibm.com/docs/en/cobol-zos/6.4?topic=reference-subscripting
But could not get to a compliable source which uses 'ALL' in table reference . Doc does say `Must be used only when the subscripted identifier is used as an intrinsic function argument or to identify a table in a format 2 SORT statement (table SORT statement).` but I couldn't figure out is meaning